client/src/main/python/ss-scale-resize.py

```diff
@@ -55,6 +55,8 @@ def _validate_and_set_scale_options(self):
 
     def doWork(self):
         super(MainProgram, self).doWork()
+        print('Requested resizing of %s' % self.node_name)
```

Comment:
After checking with @loomis. The call is asynchronous, os it's OK to leave this wording.